puzzles/signals/handlers.py

```diff
@@ -2,7 +2,7 @@
 from django.db.models.signals import pre_save, post_save, pre_delete, m2m_changed
 from django.dispatch import receiver
 from puzzles.models import Puzzle
-from puzzles.puzzle_tag import PuzzleTag, PuzzleTagColor
+from puzzles.puzzle_tag import PuzzleTag, PuzzleTagColor, META_COLOR
```

Comment:
nit: it looks like the PuzzleTagColor import is no longer used after directly importing META_COLOR

Reply:
removed it as a part of #655